python/feather/ext.pyx

```diff
@@ -24,7 +24,7 @@ from cython.operator cimport dereference as deref
 from libfeather cimport *
 
 import pandas as pd
-import pandas.core.common as pdcom
+import pandas.api.types as pdapi
```

Comment:
this is not back-compat with feather, unless the min pandas version is changed to 0.19.0 (not likely).

I might create a section in ``feather/compat.py`` where you can do something like

```
if LooseVersion(pandas.__version__) < '0.19.0'):
    pdapi = pandas.core.common
else:
    pdapi = pandas.api.dtypes
```

then ``from feather.compat import pdapi``